llvm/docs/TestingGuide.rst

```diff
@@ -529,6 +529,18 @@ RUN lines:
 
    Expands to the path separator, i.e. ``:`` (or ``;`` on Windows).
 
+``${fs-src-root}``
```

Comment:
These should be `%` not `$` but probably not worth fixing here?

Reply:
cc @keith for the doc issue here (docs [here](https://llvm.org/docs/TestingGuide.html) are wrong)